basic/exercise5/script5.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+function firstSunday() {
+    for (var year = 2014; year <= 2050; year++) {
+        var d = new Date(year, 0, 1);
+        if (d.getDay() === 0) {
+            document.getElementById("newYear").innerHTML = `The first day of ${year} is a Sunday.`
```

Comment:
This function should not touch DOM. It should return appropriate year as soon as it is found.
Function result should be assigned to DOM